packages/react-dom-factories/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "react-dom-factories",
+  "version": "16.0.0-alpha",
```

Comment:
I would think that makes sense - it was nice to have them put back in this PR though, so I can more easily cherry-pick it to the 15.6 branch. Removing can be in a follow-up that does not get cherry-picked.